crates/router/src/core/errors.rs

```diff
@@ -184,6 +184,8 @@ pub enum ConnectorError {
     },
     #[error("Invalid Configuration")]
     InvalidConnectorConfig { config: &'static str },
+    #[error("Temporary Credentials")]
+    TemporaryConnectorCredentials,
```

Comment:
This can be handled with already existing enum InvalidConnectorConfig and you can pass the proper error string in the config field